src/main/kotlin/it/pagopa/wallet/config/AzureStorageConfiguration.kt

```diff
@@ -0,0 +1,45 @@
+package it.pagopa.wallet.config
+
+import com.azure.core.http.netty.NettyAsyncHttpClientBuilder
+import com.azure.core.serializer.json.jackson.JacksonJsonSerializerBuilder
+import com.azure.core.util.serializer.JsonSerializerProvider
+import com.azure.storage.queue.QueueClientBuilder
+import com.fasterxml.jackson.databind.ObjectMapper
+import it.pagopa.wallet.client.WalletQueueClient
+import it.pagopa.wallet.config.properties.CdcQueueConfig
+import java.time.Duration
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import reactor.netty.http.client.HttpClient
+
+@Configuration
+class AzureStorageConfiguration {
+
+    @Bean
+    fun jsonSerializerProvider(objectMapper: ObjectMapper): JsonSerializerProvider =
+        JsonSerializerProvider {
+            JacksonJsonSerializerBuilder().serializer(objectMapper).build()
+        }
+
+    @Bean
+    fun expirationCdcQueueClient(
```

Comment:
Please rename in cdcQueueClient

Reply:
solved in 5d22cb3ea3f33b959620d845138a85c05e8c7975

Reply:
solved in 91dcfff58e1045f649549758a0a325dde90a46e6